src/eduid_webapp/reset_password/settings/common.py

```diff
@@ -63,3 +63,6 @@ class ResetPasswordConfig(FlaskConfig):
     chpass_timeout: int = 600
     # VCCS URL
     vccs_url: str = ''
+    # URL to get the js app that can drive the process to reset the password
+    password_reset_link: str = 'http://login.eduid.docker:8080/reset-password/'
```

Comment:
Please change password_reset_link to 'https://login.eduid.se/reset-password/', as even if it might change, it is more likely the sane default over our dev env.

Reply:
ok